src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -762,7 +762,7 @@ static bool _attrParseSvgNode(void* data, const char* key, const char* value)
     }
 #ifdef THORVG_LOG_ENABLED
     else if (!strcmp(key, "x") || !strcmp(key, "y")) {
-        if (0.0f == _parseLength(value, &type)) TVGLOG("SVG", "Unsupported attributes used [Elements type: Svg][Attribute: %s][Value: %s]", key, value);
```

Comment:
Please keep this code like this.
`if (0.0f != _parseLength(value, &type)) TVGLOG("SVG", "Unsupported attributes used [Elements type: Svg][Attribute: %s][Value: %s]", key, value);`

`==` is my fault.
The reason this condition is needed is because of the VOC(?) that, as I remember.
It generates unnecessary warnings when x and y are zero, even though the image doesn't actually change.
`x = 0`, `y = 0`, which are generated by default in certain tools, appear to have a problem even though there is no effect.

Reply:
@JSUYA could you please take a look one more time and decide, whether are we ok with the commented parts? I didn't want to remove them (yet).